### PR TITLE
Update to v0.55.0

### PR DESCRIPTION
**Pull request by phil-airship**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,15 @@
 # Zappa Changelog
 
+
+## 0.54.2
+* Support Newest Manylinux Version for dependencies (#1083)
+* Decode zappa invoke output regardless of --no-color option (#1069)
+* Convert project to using 'pipenv' to manage dependencies (#1077)
+* Ensure unique schedtuled event names (#1080)
```
Comment (hellno) [inline CHANGELOG.md:8]:
```suggestion
* Ensure unique scheduled event names (#1080)
```

**Review by hellno — COMMENTED** [review]
Looks good 🚀🙏🏼 
I just added the smallest typo fix

**Comment by phil-airship** [comment]
Updated to new minor version since I realized some of the changes could potentially break people's existing setups

**Comment by phil-airship** [comment]
Anything I can do to help get a release cut out?

**Comment by phil-airship** [comment]
Wanted to ping this again as a newer version of cryptography was published that also uses the newer Manylinux versioning scheme https://pypi.org/project/cryptography/

**Comment by javulticat** [comment]
Like most OSS, the maintainers of `Zappa` will manage the creation of new releases.  If you'd like to request a new release, please open an issue and highlight the merged PRs you'd like included in the release.  Thanks!

**Pull request closed without merging**